compiler/damlc/tests/daml-test-files/EmptyWith.daml

```diff
@@ -15,4 +13,4 @@ create = Foo with -- this should be fine
 match (Bar with) = 1 -- this should be fine
 
 update : Bar -> Bar
-update x = x with -- this should be an error (according to Haskell)
+update x = x with -- this should be be fine (since record dot updates)
```

Comment:
is this also accepted in GHC? I don’t really get why record dot updates change whether an empty record update is valid or not.